code/log/lib/Loot/Log/Internal/Name.hs

```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE TypeFamilies #-}
 module Loot.Log.Internal.Name
        ( Name (Name, unName)
-       , fromList
+       , GHC.Exts.fromList
```

Comment:
Fixed :ok_hand: 

Btw, is it correct that you meant that doing `import hiding` is better **only in this module** because we already hide `toList`, and it's ok to leave qualified `toList` name in other modules?

Reply:
Yep, that's correct